streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -2421,7 +2421,7 @@ final class ZStream[-R, +E, +A] private (val channel: ZChannel[R, Any, Any, Any,
    */
   def refineOrDie[E1](
     pf: PartialFunction[E, E1]
-  )(implicit ev1: E <:< Throwable, ev2: CanFail[E], trace: Trace): ZStream[R, E1, A] =
+  )(implicit ev1: E IsSubtypeOfError Throwable, ev2: CanFail[E], trace: Trace): ZStream[R, E1, A] =
```

Comment:
This change is not going to be binary compatible.

Reply:
Fixed